Packs/CommonScripts/Scripts/GetTime/README.md

```diff
@@ -1,30 +1,36 @@
-Retrieves the current date and time. 
+Retrieves the current date and time.
+
 
 ## Script Data
+
 ---
 
 | **Name** | **Description** |
 | --- | --- |
 | Script Type | javascript |
 | Tags | Utility |
-
+| Cortex XSOAR Version | 5.0.0 |
 
 ## Inputs
+
 ---
 
 | **Argument Name** | **Description** |
 | --- | --- |
 | date | The date to use in the object. |
-| dateFormat | The date format for the result. Can be "ISO", "GMT", "UTC", "Locale", "Date", "Year", "Month", "Day", "DayInWeek", "Hours". |
-| contextKey | The prefix the keys in the context for the results. |
-| monthsAgo | Subtracts monthsAgo months from current time. |
-| minutesAgo | Subtracts minutesAgo minutes from current time. |
-| hoursAgo | Subtracts hoursAgo hours from current time. |
+| dateFormat | The date format for the result, can be ISO/GMT/UTC/Locale/Date/Year/Month/Day/DayInWeek/Hours.<br/>Note that the response will start from 0, E.G. if the current month is October<br/>\(the 10th month\) the result will be 9.<br/>Note:<br/>In XSOAR versions 6.11.0 or later, or 8.1.0 or later, the time format returned when UTC or GMT is given has changed.<br/>Instead of 'Wed Jan 18 2023 12:59:12 GMT\+0000 \(UTC\)', The output will be 'Wed, 18 Jan 2023 15:04:41 GMT'. |
```

Comment:
```suggestion
| dateFormat | The date format for the result. Can be ISO/GMT/UTC/Locale/Date/Year/Month/Day/DayInWeek/Hours.<br/>Note that the response will start from 0. For example, if the current month is October<br/>\(the 10th month\) the result will be 9.<br/>Note:<br/>In Cortex XSOAR versions 6.11 or later, or 8.1.0 or later, the time format returned when UTC or GMT is given has changed.<br/>Instead of 'Wed Jan 18 2023 12:59:12 GMT\+0000 \(UTC\)', The output will be 'Wed, 18 Jan 2023 15:59:12 GMT'. |
```